CollapseLauncher/Classes/GameManagement/GameSettings/Universal/RegistryClass/CollapseScreenSetting.cs

```diff
@@ -28,6 +28,12 @@ internal class CollapseScreenSetting : IGameSettingsValue<CollapseScreenSetting>
         /// </summary>
         public bool UseExclusiveFullscreen { get; set; } = false;
 
+        /// <summary>
+        /// This defines if the game gonna be running with Borderless option enabled.<br/><br/>
```

Comment:
Nitpick but you can write "going to be" instead of "gonna" XD

Reply:
fixed